shadowsocks-csharp/View/ConfigForm.cs

```diff
@@ -79,7 +79,14 @@ private static void Init()
             public static EncryptionMethod GetMethod(string name)
             {
                 if (!init) Init();
-                return methodByName[name];
+                bool ok = methodByName.TryGetValue(name, out EncryptionMethod method);

```

Comment:
bool success = ...

Reply:
Done.

By the way, seems NLog cause some strange deadlock in ShadowsocksController.Reload(). Open ConfigForm, click OK will trigger it.

Reply:
There was a log output file issue when opening config form. Fixed in recent commits.